packages/interval/package.json

```diff
@@ -41,6 +42,7 @@
     "react": "^15.3.0 || ^16.0.0"
   },
   "dependencies": {
+    "@types/react": "^16.0.2",
```

Comment:
should it go to dev deps?

Reply:
This react typings should be installed when other people will install this package.

Reply:
I'm not sure but maybe i should replace it to peer dependency

Reply:
So probably peerDeps then. We do not use TS and have no plans to, so installing unnecessary dependency does not seem quite right.

Reply:
Or even `optionalDependencies` would be better I reckon
The reason why it is not good to add such things into main deps is that they will live in a project after `yarn install --production` and will end up in docker images and also in licence reports...

Reply:
No, optional dependencies is not correct, because React typings is using inside this typings.

Reply:
I'm surprised how many people put `@types/*` into their production deps 
https://github.com/search?utf8=✓&q=%27%40types%2Freact%27+extension%3Ajson&type=Code

This is quite sad tbh